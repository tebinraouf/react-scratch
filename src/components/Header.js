import React from "react";
import { NavLink } from "react-router-dom";
import "../static/styles/header.css";

const Header = () => (
	<div>
        This is the header.
		<nav>
			<NavLink to="/">
                Home
			</NavLink>
			<NavLink to="/about">
                About
			</NavLink>
			<NavLink to="/movie/0">
                Get by URL Parameter
			</NavLink>
		</nav>
	</div>
);

export default Header;