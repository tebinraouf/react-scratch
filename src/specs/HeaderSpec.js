import React from "react";
import { BrowserRouter } from "react-router-dom";
import Header from "../components/Header";
import renderer from "react-test-renderer";

describe("<Header />", () => {
	test("should render correctly", () => {
		const component = renderer.create(
			<BrowserRouter>
				<Header />
			</BrowserRouter>
		);
		const tree = component.toJSON();

		expect(tree).toMatchSnapshot();
	});
});