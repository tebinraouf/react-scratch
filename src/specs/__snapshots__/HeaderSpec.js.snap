// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render correctly 1`] = `
<div>
  This is the header.
  <nav>
    <a
      aria-current="page"
      className="active"
      href="/"
      onClick={[Function]}
      style={Object {}}
    >
      Home
    </a>
    <a
      aria-current={null}
      href="/about"
      onClick={[Function]}
    >
      About
    </a>
    <a
      aria-current={null}
      href="/movie/0"
      onClick={[Function]}
    >
      Get by URL Parameter
    </a>
  </nav>
</div>
`;
